solutions/account-infrastructure-base/main.tf

```diff
@@ -2,6 +2,167 @@
+      global_deny      = false
+    }
+    "databases-for-elasticsearch" = {
+      enforcement_mode = var.cbr_enforcement_mode == "report" ? "disabled" : var.cbr_enforcement_mode
+      global_deny      = false
+    }
+    "databases-for-enterprisedb" = {
+      enforcement_mode = var.cbr_enforcement_mode == "report" ? "disabled" : var.cbr_enforcement_mode
+      global_deny      = false
+    }
+    "databases-for-etcd" = {
+      enforcement_mode = var.cbr_enforcement_mode == "report" ? "disabled" : var.cbr_enforcement_mode
+      global_deny      = false
+    }
+    "databases-for-mongodb" = {
+      enforcement_mode = var.cbr_enforcement_mode == "report" ? "disabled" : var.cbr_enforcement_mode
+      global_deny      = false
+    }
+    "databases-for-mysql" = {
+      enforcement_mode = var.cbr_enforcement_mode == "report" ? "disabled" : var.cbr_enforcement_mode
+      global_deny      = false
+    }
+    "databases-for-postgresql" = {
+      enforcement_mode = var.cbr_enforcement_mode == "report" ? "disabled" : var.cbr_enforcement_mode
+      global_deny      = false
+    }
+    "databases-for-redis" = {
+      enforcement_mode = var.cbr_enforcement_mode == "report" ? "disabled" : var.cbr_enforcement_mode
+      global_deny      = false
+    }
+    "directlink" = {
+      enforcement_mode = var.cbr_enforcement_mode
+      global_deny      = false
+    }
+    "dns-svcs" = {
+      enforcement_mode = var.cbr_enforcement_mode
+      global_deny      = false
+    }
+    "event-notifications" = {
+      enforcement_mode = var.cbr_enforcement_mode == "report" ? "disabled" : var.cbr_enforcement_mode
+      global_deny      = false
+    }
+    "globalcatalog-collection" = {
+      enforcement_mode = var.cbr_enforcement_mode
+      global_deny      = false
+    }
+    "hs-crypto" = {
+      enforcement_mode = var.cbr_enforcement_mode
+      global_deny      = false
+    }
+    "iam-access-management" = {
+      enforcement_mode = var.cbr_enforcement_mode
+      global_deny      = false
+    }
+    "iam-groups" = {
+      enforcement_mode = var.cbr_enforcement_mode
+      global_deny      = false
+    }
+    "iam-identity" = {
+      enforcement_mode = var.cbr_enforcement_mode
+      global_deny      = false
+    }
+    "is" = {
+      enforcement_mode = var.cbr_enforcement_mode
+      global_deny      = false
+    }
+    "kms" = {
+      enforcement_mode = var.cbr_enforcement_mode
+      global_deny      = false
+    }
+    "logdna" = {
+      enforcement_mode = var.cbr_enforcement_mode
+      global_deny      = false
+    }
+    "logdnaat" = {
+      enforcement_mode = var.cbr_enforcement_mode
+      global_deny      = false
+    }
+    "messagehub" = {
+      enforcement_mode = var.cbr_enforcement_mode
+      global_deny      = false
+    }
+    "messages-for-rabbitmq" = {
+      enforcement_mode = var.cbr_enforcement_mode == "report" ? "disabled" : var.cbr_enforcement_mode
+      global_deny      = false
+    }
+    "mqcloud" = {
```

Comment:
Need to remove this service as this is only supported in eu-fr2 region for which either region or service instance id is mandatory to provide, mentioned here [Note on mqcloud](https://github.com/terraform-ibm-modules/terraform-ibm-cbr/tree/main/modules/fscloud).